### PR TITLE
BJLS and BEF import projects consolidation

### PR DESCRIPTION
**Pull request by AlexeiShmidman**

### CODE REVIEW

**Comment by plaird** [comment]
I will try to review tonight. Thanks!

**Comment by plaird** [comment]
Looks like there are Windows test failures:

Failures: 
  BazelProjectImporterTest.basic:92 expected:<2> but was:<0>
  BazelProjectImporterTest.withQueryInTargetFile:135 expected:<1> but was:<0>
  BazelProjectImporterTest.withSubpackage:111 expected:<1> but was:<0>

I can help troubleshoot if you need (my son has a Windows laptop). This is almost always a path delimiter issue.

**Comment by AlexeiShmidman** [comment]
> Looks like there are Windows test failures:
> 
> Failures: BazelProjectImporterTest.basic:92 expected:<2> but was:<0> BazelProjectImporterTest.withQueryInTargetFile:135 expected:<1> but was:<0> BazelProjectImporterTest.withSubpackage:111 expected:<1> but was:<0>
> 
> I can help troubleshoot if you need (my son has a Windows laptop). This is almost always a path delimiter issue.

Thanks a lot! will try to detect the issue place by myself and if I face any problem will ask you

File: bundles/com.salesforce.bazel-java-sdk/src/main/java/com/salesforce/bazel/sdk/command/BazelWorkspaceCommandRunner.java [hunk 1]
```diff
@@ -599,7 +599,9 @@ public File getBazelGeneratedFilesFolder() {
     public String getProjectOutputPath(BazelLabel bazelLabel) {
         // bazel-bin/[PACKAGE_PATH]/_javac/[TARGET_NAME]/[TARGET_TYPE][LAST_PATH_COMP_TARGET_NAME]_classes
         File genFolder = getBazelGeneratedFilesFolder();
-        File javacFolder = new File(genFolder, bazelLabel.getPackageName() + "/_javac/" + bazelLabel.getTargetName());
+        String pathToClasses =
```
Comment (plaird) [inline bundles/com.salesforce.bazel-java-sdk/src/main/java/com/salesforce/bazel/sdk/command/BazelWorkspaceCommandRunner.java:602]: Aha! :)

**Review by plaird — APPROVED** [review]

**Review by kkazakevich — APPROVED** [review]

**Pull request merged**